### DISCUSSION TITLE
Demo Mode options input

### DISCUSSION THREAD

**Discussion by SabreKnot** [ikemen-engine/Ikemen-GO#1302] [Feature Request] [answered]
I have noticed that on IKEMEN the Demo Mode has the Round 1 & Fight Animation included. Where as on Mugen 1.0/1.1 does not (it only shows the combo counter) Are you able to please include toggle 
on/off options in the systembase also for Round 1 / Fight Animations & Combo Counter display?

Please include this in a future update! 
Thank you.

**Comment by RealFoobs** [comment]
While on the surface this seems a minor and purely aesthetic incompatibility, it might be a sign of bigger discrepancies between how round states are processed between engines. I vaguely remember PotS saying they aren't quite 1:1

**Comment by potsmugen** [comment, marked as answer]
A lot of what happens in Demo mode is now exposed in `demo.zss`. Round calls are among the things that can be edited.
